core-library/src/main/java/org/silverpeas/core/calendar/ical4j/ICal4JRecurrenceCodec.java

```diff
@@ -188,43 +182,28 @@ public Recurrence decode(final VEvent vEvent, final ZoneId defaultZoneId) {
     if (recur.getCount() > 0) {
       recurrence.until(recur.getCount());
     } else if (recur.getUntil() != null) {
-      Temporal temporalUntil = iCal4JDateCodec.decode(recur.getUntil(), defaultZoneId);
+      Temporal temporalUntil = iCal4JDateCodec.decode(recur.getUntil());
       recurrence.until(temporalUntil);
     }
     if (recur.getDayList() != null && !recur.getDayList().isEmpty()) {
       recurrence.on(recur.getDayList().stream().map(this::decode).collect(Collectors.toList()));
     }
-    processExclusionDates(vEvent, recurrence, defaultZoneId);
+    processExclusionDates(vEvent, recurrence);
     return recurrence;
   }
 
-  private void processExclusionDates(final VEvent vEvent, final Recurrence recurrence,
-      final ZoneId defaultZoneId) {
+  private void processExclusionDates(final VEvent vEvent, final Recurrence recurrence) {
     /* TODO activating following commented forEach after fix of EXDATE UTC management (iCal4J)
        cf. https://github.com/ical4j/ical4j/issues/113 for example
 
        <code>exDates.forEach(exDate -> recurrence
-          .excludeEventOccurrencesStartingAt(iCal4JDateCodec.decode(exDate, defaultZoneId)));</code>
+          .excludeEventOccurrencesStartingAt(iCal4JDateCodec.decode(exDate)));</code>
 
       Deleting following forEach after fix of EXDATE UTC management (iCal4J)
       cf. https://github.com/ical4j/ical4j/issues/113 for example
       */
```

Comment:
Did you check the bug is always here?

Reply:
The issue is still open.

Reply:
Yes on Github. But that doesn't mean it isn't fixed (by another way)

Reply:
I will spend time on this.
As the Silverpeas's code can not be removed in any case in order to handle exlusion dates, and also as the code become simplier, there is no more reason finally to keep this comment.

Reply:
Removed.